lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -240,7 +238,7 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
       data-display-name="${_("Discussion Moderators")}"
       data-info-text="
         ${_("Discussion Moderators can edit or delete any post, clear misuse flags, close "
-         "and re-open threads, endorse responses, and see posts from all cohorts. "
+         "and re-open threads, endorse responses, and see posts from all groups. "
```

Comment:
"You can only give course team roles to enrolled users." -> "Only enrolled users can be added as Discussion Moderators."